src/assets/script.js

```diff
@@ -5,22 +5,29 @@
 /* Cookie consent banner customization */
 (function () {
   window.addEventListener('tac.root_available', function (e) {
-    document.getElementById('tarteaucitronPersonalize').classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
-    document.getElementById('tarteaucitronAllAllowed').classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
-    
-    document.getElementById('tarteaucitronCloseAlert').classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
-    document.getElementById('tarteaucitronAllDenied').classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
+
+    document.querySelectorAll('#tarteaucitronRoot button').forEach(function (button) {
+      if (button.classList.contains('catToggleBtn') || button.id === 'tarteaucitronClosePanel') {
+        return
+      }
+
+      if (button.classList.contains('tarteaucitronAllow')) {
+        button.classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
+      } else {
+        button.classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
+      }
+    })
   }, { once: true })
 
   window.addEventListener('tac.open_panel', function () {
     const servicesContainer = document.getElementById('tarteaucitronServices_api')
 
-    servicesContainer.querySelectorAll('.tarteaucitronAllow').forEach(function (button) {
-      button.classList.add('btn', 'btn-primary', 'ml-2')
-    })
-
-    servicesContainer.querySelectorAll('.tarteaucitronDeny').forEach(function (button) {
-      button.classList.add('btn', 'btn-secondary', 'ml-2')
+    servicesContainer.querySelectorAll('button').forEach(function (button) {
+      if (button.classList.contains('tarteaucitronAllow')) {
+        button.classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
+      } else {
+        button.classList.add('btn', 'btn-secondary', 'btn-info', 'ml-2')
```

Comment:
Setting both `.btn-secondary` and `.btn-info` is redundant here: did you mean `.btn-inverse` maybe?

Reply:
In fact, all I needed was `.btn-info` here, thanks!